triage_bot/lib/discord.dart

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:nyxx/nyxx.dart';
```

Comment:
I can see this being useful in a separate lib.

Reply:
which part? the boilerplate stuff (constants and stripBoilerplate method) are extremely specific to this bot, and the sendDiscordMessage is really barely more than a wrapper around nyxx intended to make some of the messages the bot sends more consistent. I'm not sure it's really valuable for other projects? The nyxx API itself is pretty straight-forward.